sale_product_set/wizard/product_set_add.py

```diff
@@ -10,10 +10,12 @@ class ProductSetAdd(models.TransientModel):
     _rec_name = "product_set_id"
     _description = "Wizard model to add product set into a quotation"
 
+    required_field = fields.Selection(
```

Comment:
Don't do this on your inherited module, but inherit the wizard model with other name, and there, put `order_id` as required=False, and the new m2o as required=True